test/e2e/tests/single_zone_e2e_test.go

```diff
@@ -0,0 +1,113 @@
+/*
+Copyright 2018 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package tests
+
+import (
+	"fmt"
+	"path/filepath"
+	"strings"
+
+	"k8s.io/apimachinery/pkg/util/uuid"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	testutils "sigs.k8s.io/gcp-filestore-csi-driver/test/e2e/utils"
+	remote "sigs.k8s.io/gcp-filestore-csi-driver/test/remote"
+)
+
+const (
+	testNamePrefix = "gcfs-csi-e2e-"
+
+	defaultSizeGb int64 = 5
+)
+
+var _ = Describe("GCFS CSI Driver", func() {
+
+	It("Should create->attach->stage->mount volume and check if it is writable, then unmount->unstage->detach->delete and check disk is deleted", func() {
```

Comment:
remove "attach", "stage", "unstage" and "detach" since filestore doesn't implement those operations